### ISSUE TITLE
Need Documentation

### ISSUE THREAD

**Issue by krisanalfa** [xinix-technology/yesbee#5]
Hi, Sir. I really glad if there's documentation about it's fundamental and how to use it. Thank you for your great job!

**Comment by reekoheek** [comment]
Yeah, I've planned to do that. Since I'm a single maintainer for `yesbee` (and some others projects), It is hard for me to do that. I've managed my time to do that in some spare time between clients projects that used `yesbee`.

So what do you want to use `yesbee` for? Are you willing to contribute?

Sorry for the trouble that raised caused by no documentation available.

**Comment by krisanalfa** [comment]
Ah, I understand. So maybe you need to publish this package, start to make a screencast or blog post. So the others aware and use `yesbee`, from that we can gain contributors. Sir, I really recommend you to write documentation so you won't maintain this repo only on yourself.

Yes I want to use it for my next project. I only want to know the basic. I want to use it for `bootstrapping` my project. Like what [Yeoman](http://yeoman.io/) did. **I want to know what features does `yesbee` have.**

Only if I have a clue how it works, how it's philosophy, and how it's fundamental, I will start to watch and contribute. But from now on, I will try to understand it and write a blog post on how to use `yesbee`.
